### PR TITLE
Update policy-csp-bitlocker.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #2145,
added footnote

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
added link to bitlocker CSP

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/policy-csp-bitlocker.md [hunk 1]
```diff
@@ -83,12 +83,17 @@ The following list shows the supported values:
 <!--/Policy-->
 <hr/>
 
+> [!NOTE]
+> To manage encryption of PCs and devices, use [BitLocker CSP](https://docs.microsoft.com/windows/client-management/mdm/bitlocker-csp)
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-bitlocker.md]:
```suggestion
> To manage encryption of PCs and devices, use [BitLocker CSP](https://docs.microsoft.com/windows/client-management/mdm/bitlocker-csp).
```

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review made by JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Pull request merged**